### PR TITLE
build: include scss imports within babel compiled output

### PR DESCRIPTION
**Pull request by priyajeet**
BREAKING CHANGE: The babel compiled output will now include SCSS imports. This commit removes the `babel-plugin-transform-require-ignore` which was responsible for removing lines such as `import './ContentPreview.scss'` from the babel compiled version of `ContentPreview.js`. This forced customers to manually include styles via `import 'box-ui-elements/dist/preview.css'`. With this change, the babel compiled output will now include `import './ContentPreview.scss'` and no manual import of `preview.css` will be required. This change however requires that [webpack is configured properly](https://github.com/box/box-ui-elements#stylesheets) to handle `.scss` files. As a customer, you will notice the change as shown below:

**Before**
-----------
Notice the import of `preview.css`.
```jsx
import React from 'react';	
import { render } from 'react-dom';	
import { ContentPreview } from 'box-ui-elements';	
import messages from 'box-ui-elements/i18n/en-US';	
import 'box-ui-elements/dist/preview.css';

render(	
    <ContentPreview	
        fileId='FILE_ID'	
        token='ACCESS_TOKEN'	
        language='en-US'	
        messages={messages}	
    />,	
    document.querySelector('.container')	
);	
```

**After**
--------
Notice no import of `preview.css`.

```jsx
import React from 'react';	
import { render } from 'react-dom';	
import { ContentPreview } from 'box-ui-elements';	
import messages from 'box-ui-elements/i18n/en-US';

render(	
    <ContentPreview	
        fileId='FILE_ID'	
        token='ACCESS_TOKEN'	
        language='en-US'	
        messages={messages}	
    />,	
    document.querySelector('.container')	
);	
```

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @priyajeet has signed the CLA. Thanks for the pull request!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ longer description here if necessary.
 include BREAKING CHANGE keyword for breaking changes.
 ```
 
-The message summary should be a one-sentence description of the change, and it must be 72 characters in length or shorter. For a list of tags, please [click here](https://github.com/marionebl/commitlint/tree/master/%40commitlint/config-conventional#type-enum). We use the angular style for tags. See the [default release rules](https://github.com/semantic-release/commit-analyzer/blob/master/lib/default-release-rules.js) based on the commit tag. Shown below is an example of the release type that will be done based on a commit message.
+The message summary should be a one-sentence description of the change, and it must be 72 characters in length or shorter. For a list of tags, please [click here](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional#type-enum). We use the angular style for tags. See the [default release rules](https://github.com/semantic-release/commit-analyzer/blob/master/lib/default-release-rules.js) based on the commit tag. Shown below is an example of the release type that will be done based on a commit message.
```
Comment (priyajeet) [inline CONTRIBUTING.md]: marionebl --> conventional-changelog

File: package.json [hunk 2]
```diff
@@ -66,7 +66,7 @@
         "start": "yarn start:examples",
         "start:dev": "yarn setup; LANGUAGE=en-US BABEL_ENV=dev NODE_ENV=dev node --max_old_space_size=8192 node_modules/webpack-dev-server/bin/webpack-dev-server.js --config conf/webpack.config.js --mode development",
         "start:examples": "yarn setup; EXAMPLES=true LANGUAGE=en-US REACT=true BABEL_ENV=dev NODE_ENV=dev node --max_old_space_size=8192 node_modules/react-styleguidist/bin/styleguidist.js server --config conf/styleguide.config.js --mode development",
-        "start:npm": "yarn setup; npm-run-all build:prod:npm build:dev:es",
+        "start:npm": "yarn setup; yarn build:dev:es",
```
Comment (priyajeet) [inline package.json]: No need to build CSS files via webpack anymore for dev linking.
Leaving it for live, since they are harmless.

**Pull request merged**